.github/workflows/ci-e2e.yml

```
@@ -39,7 +39,7 @@ jobs:
           parallel: true
           build: npm run cypress:build
           start: npm run start
-          wait-on: 'http://localhost:3000/prices'
+          wait-on: 'http://localhost:3000/oracles'
```

```suggestion
          wait-on: 'http://localhost:3000'
```

I think can remove `/prices` since we have a homepage now